extension/chrome/src/js/storage.js

```diff
@@ -27,196 +27,247 @@ var extensionStorage = (function() {
 		checkversion: true
 	};
 
-	var cloudStorage = chrome.storage.sync;
+	let cloudStorage = chrome.storage.sync;
```

Comment:
There are more parts in new FF that were not available two years ago (like browser.runtime.onMessageExternal) so more workarounds can be removed. I prefer to do that in some other PR